packages/components/src/modifiers/hds-tooltip.ts

```diff
@@ -0,0 +1,213 @@
+  ): Partial<TippyProps> {
+    const { options } = named;
+    let [content] = positional;
+
+    let $anchor: HTMLElement | null = element; // Ensure $anchor can be null
+
+    if (typeof options?.triggerTarget === 'string') {
+      const $el = $anchor;
+      if (options.triggerTarget === 'parentNode') {
+        if ($anchor.parentNode instanceof HTMLElement) {
+          // Type guard
+          $anchor = $anchor.parentNode;
+        }
+      } else {
+        const queryResult: HTMLElement | null = $anchor.querySelector(
+          options.triggerTarget
+        );
+        if (queryResult) {
+          $anchor = queryResult;
+        }
+      }
+      if ($anchor instanceof HTMLElement) {
+        // Ensure $anchor is an HTMLElement before cloning
+        const clonedElement = $anchor.cloneNode(true) as HTMLElement; // Explicitly cast cloned node to HTMLElement
+        content = clonedElement.outerHTML; // Now safely access outerHTML
+      }
+      $el?.remove(); // Use optional chaining in case $el is null
+      options.triggerTarget = null;
+    }
+
+    // The {{hds-tooltip}} will just use the HTML content.
+    if (typeof content === 'undefined' && $anchor instanceof HTMLElement) {
+      // Ensure $anchor is an HTMLElement before accessing innerHTML
+      content = $anchor.innerHTML;
+      $anchor.innerHTML = '';
+    }
+
+    if (options?.trigger === 'manual') {
+      // If we are manually triggering, a out delay means only show for the
+      // amount of time specified by the delay.
+      const delay = options.delay || [];
+
+      if (Array.isArray(delay) && delay.length) {
+        if (typeof delay[1] !== 'undefined') {
+          options.onShown = (tooltip) => {
+            clearInterval(this.interval);
+            this.interval = setTimeout(() => {
+              tooltip.hide();
+            }, delay[1] ?? 0);
+          };
+        }
+      }
+    }
+
+    const $trigger = $anchor;
+
+    if (!$trigger.hasAttribute('tabindex')) {
+      this.needsTabIndex = true;
+      $trigger.setAttribute('tabindex', '0');
+    }
+
+    /*  Typescript does not like the previous approach of adding an undefined value
+     **  to the array and then filtering it out.
+     */
+    const plugins =
+      options?.followCursor !== undefined
+        ? [this.hideOnEsc, followCursor]
+        : [this.hideOnEsc];
+
+    return {
+      theme: 'hds',
+      triggerTarget: $trigger,
+      arrow: `
+        <svg class="hds-tooltip-pointer" width="16" height="7" viewBox="0 0 16 7" xmlns="http://www.w3.org/2000/svg">
+          <path d="M0 7H16L9.11989 0.444571C8.49776 -0.148191 7.50224 -0.148191 6.88011 0.444572L0 7Z" />
+        </svg>`,
+      // keeps tooltip itself open on hover:
+      interactive: true,
+      // fix accessibility features that get messed up with setting interactive: true
+      aria: <TippyProps['aria']>{
+        content: 'describedby',
+        expanded: false,
+      },
```

Comment:
suggestion: from my quick tests, we don't need to type the aria property
```suggestion
      aria: {
        content: 'describedby',
        expanded: false,
      },
```

Reply:
done